_includes/header.html

```diff
@@ -1,17 +1,17 @@
-<header class="pldoc-header-site">
+<header class="pldoc-header-site" role="banner" aria-labelledby="pldoc_header_site_title">
     {% if page.title == "Home" %}
-    <h1 class="pldoc-header-site-title">
+    <h1 class="pldoc-header-site-title" id="pldoc_header_site_title">
         <a class="pldoc-link title-name" href="{{ site.baseurl }}">
             {{ site.title }}
         </a>
-        <span class="title-description copy copy-base">{{ site.tagline }}</span>
     </h1>
+    <span class="title-description copy copy-base">{{ site.tagline }}</span>
```

Comment:
Since this is now outside of the ``pldoc-header-site-title`` element, can you add a prefix of ``pl-`` to the class name?